cmd/minikube/cmd/config/util_test.go

```diff
@@ -79,3 +80,15 @@ func TestSetBool(t *testing.T) {
 		t.Fatalf("SetBool set wrong value")
 	}
 }
+
+func TestIsAddonAlreadySet(t *testing.T) {
+	names := []string{"ingress", "heapster"}
+	for _, name := range names {
+		addon := assets.Addons[name]
+		addonStatus, _ := addon.IsEnabled()
+		err := isAddonAlreadySet(addon, addonStatus)
+		if err == nil {
+			t.Fatalf("did not return error for the addon was already set")
```

Comment:
Thanks for adding the unit tests, two things to impriove this unit test
1- first lets change the t.Fatalf to t.Error
2- lets make the message in the following format:
```
t.Errorf("Expected to get error , got nil", )
```

Reply:
Done. Thanks